<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkDepositScreen</class>
 <widget class="QMainWindow" name="checkDepositScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="checkDepositText">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Deposit Check</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 15px;
}

QPushButton:hover {
    border: 1px solid white;
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkNumberEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Check Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkAmountEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Check Amount</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="depositCheckingButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Checking</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="depositSavingsButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Savings</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkNumberLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Enter Check Number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="checkAmountLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Enter Check Amount:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dollarSignLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>$</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
